doc/getting_started/local_installation.md

```diff
@@ -24,6 +24,15 @@ $ brew tap pachyderm/tap && brew install pachyderm/tap/pachctl@1.6
 
 # For Linux (64 bit):
 $ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.6/pachctl_1.6.6_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
+
+# For Windows10: enable Windows Subsystem for Linux by running this on Win shell & Restart the system.This will
```

Comment:
Thanks! I think this could be a little more concise and combine with the Linux command. Maybe:

```
# For Linux (64 bit) or Window 10+ in WSL:
$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.6/pachctl_1.6.6_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
```